### DISCUSSION TITLE
Font Type, Color etc

### DISCUSSION THREAD

**Discussion by 24se7en** [Cuperino/QPrompt-Teleprompter#209] [General]
It would be great if i can change the font type, height and color of several word(s) like on https://bigstageteleprompter.online/

**Comment by Cuperino** [comment]
Hi @24se7en,

Do you mean have these changes affect all of the script instead of only selected parts?  I'm asking because QPrompt already allows 
you to do these things all of these things, but you must select the text you wish to edit first.  You can select all text by pressing Ctrl-A or Command-A on Mac, like in most text editors.

**Comment by 24se7en** [comment]
P.J.A. van den Berg
Studio Eleven Autocue Systems
Rijersstraat 12
NL 5708 KP Helmond
Telefoon: +31 492 83 3315
Mobile +31 621 91 6570

E-mail:  ***@***.******@***.***> ***@***.***

KvK : 75541084
VAT ID: NL001289388B79

 

Van: Javier O. Cordero Pérez ***@***.***> 
Verzonden: maandag 21 augustus 2023 17:38
Aan: Cuperino/QPrompt ***@***.***>
CC: 24se7en ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [Cuperino/QPrompt] Font Type, Color etc (Discussion #209)

 

Hi @24se7en <https://github.com/24se7en> ,

Do you mean have these changes affect all of the script instead of only selected parts? I'm asking because QPrompt already allows
you to do these things all of these things, but you must select the text you wish to edit first You can select all text by pressing Ctrl-A or Command-A on Mac, like in most text editors.

**Reply by Cuperino** [reply]
This is strange... Something went wrong with your reply and the contents of your reply are missing. Please try to reply using the web interface instead:
https://github.com/Cuperino/QPrompt/discussions/209

**Reply by 24se7en** [reply]
I hope you get them now

**Comment by 24se7en** [comment]
![2023-08-21_18-52-46](https://github.com/Cuperino/QPrompt/assets/128812814/df3af897-10e7-4944-ab70-3f5dc6997a25)
![2023-08-21_15-25-30](https://github.com/Cuperino/QPrompt/assets/128812814/c9804c53-6502-418b-9cc8-5dcba4a021f2)

**Reply by 24se7en** [reply]
i click with my right mouse button and select Font but nothing happened

**Reply by Cuperino** [reply]
You've found a bug!

Unfortunately I'm no longer able to build v1.1.x binaries without introducing a bug related to full screen mode, because of changes in our build pipeline outside of my control. I'll implement the fix for this in the both the 1.1.x and 1.2.0 branches, but only make binaries available for the v1.2 branch, because it is okay to have the full screen bug in those early builds.

In the meantime, you can still open the fonts selection menu by clicking on the `A` icon at the editor's toolbar, and then on `Choose System Font`. I will update here when the issue is fixed, and after when it's made available in builds.

**Reply by Cuperino** [reply]
Issue filed at: https://github.com/Cuperino/QPrompt/issues/210

**Comment by 24se7en** [comment]
Where i can select : and then on Choose System Font?

**Comment by Cuperino** [comment]
1. Select the text you wish to edit
![screenshot of text selection](https://github.com/Cuperino/QPrompt/assets/1840407/109b4068-4020-4312-a8ff-447d806795cb)

2. Click on the `A` button in the editor toolbar.
![screenshot of the A button](https://github.com/Cuperino/QPrompt/assets/1840407/57ae5489-fe78-41fe-8de4-6fa7833e01df)

3. Then click on `Choose System Font`
![screenshot of the font selection menu](https://github.com/Cuperino/QPrompt/assets/1840407/bd8c2510-4c3d-4cab-843c-fb9f2605e3be)

4. Select system font from the font selection dialog
![image](https://github.com/Cuperino/QPrompt/assets/1840407/e92ffaf8-dbcf-4504-9fe7-1cf072020a61)

**Comment by 24se7en** [comment]
HI Cuperino,

Sorry but this menu i dont have in my Qprompt 2.0

**Comment by Cuperino** [comment]
All versions of QPrompt since [1.0.0-beta-05](https://github.com/Cuperino/QPrompt/releases/tag/v1.0.0-beta-005) have this menu. The current system font selection dialog was introduced in [1.1.3](https://github.com/Cuperino/QPrompt/releases/tag/v1.1.3).

There's no QPrompt 2.0 at the time of writing, you may be thinking of the 1.2.0 branch. The newest versions of QPrompt are 1.1.6 on the public channels (which are [SourceForge](https://sourceforge.net/projects/qprompt/), [GitHub releases](https://github.com/Cuperino/QPrompt/releases/tag/v1.1.6), [Snapcraft](https://snapcraft.io/qprompt), [Flathub](https://flathub.org/apps/com.cuperino.qprompt)) and 1.2.0-early-access-06 on the early access beta channel (which is [Patreon](https://www.patreon.com/posts/early-access-v1-86901646)). Other distribution channels (such as AUR, and AppImageHub) are managed by third parties and are, therefore, unsupported. I would still investigate what the issue is, but no solution would be provided through those channels.

- What Operating System are you using QPrompt on? (Android and HaikuOS do not offer the option to choose from system fonts)
- Where did you source your installation of QPrompt?
